packages/storybook/src/react/Tabs.stories.tsx

```diff
@@ -1,6 +1,8 @@
 import { Meta, StoryObj } from '@storybook/react';
 import { Tabs, TabsProps } from '@gemeente-denhaag/tab';
 import readme from '../../../../components/Tab/README.md';
+import { useEffect, useState } from 'react';
```

Comment:
Ik heb persoonlijk een voorkeur voor alle imports van 1 package in 1 import statement te doen zoals hier: `import React, { useEffect, useState } from 'react';`.

Reply:
Eens, normaal doet de autocomplete dat ook maar nu ff niet...